hooks-admin.php

```diff
@@ -2,9 +2,7 @@
 
 // Filter export formats array
 function pb_mpdf_add_to_formats( $formats ) {
-		unset( $formats['standard']['pdf'] );
-		unset( $formats['standard']['print_pdf'] );
-		$formats['standard'] = array( 'mpdf' => __( 'PDF (mPDF)', 'pressbooks' ) ) + $formats['standard'];
+		array_splice( $formats['standard'], 2, 0, array( 'mpdf' => __( 'PDF (mPDF)', 'pressbooks' ) ) );
```

Comment:
Yes, I remember running into this before — `PB_PLUGIN_VERSION` and other PB constants are not available to plugins outside of PB unless they include the `pressbooks.php` file in their plugin. Since a link to '/pressbooks/compatibility.php` is already required, from an external plugin's perspective it may be more straight forward to have the constant available there.